native_client/client.cc

```diff
@@ -200,7 +200,7 @@ main(int argc, char **argv)
     return 1;
   }
 
-  if (lm && trie) {
+  if (lm) {
```

Comment:
What is the runtime impact of trie being built on the fly ?

Reply:
It makes start-up time *really* slow. I expect the main (only?) use case of not passing a trie is when you're experimenting with language models, so you can postpone the task of generating all the trie files.

Reply:
What I do worry about is that people won't carefully use it and we will have a bunch of issues about how it is slow.

Reply:
We're already seeing such things where people do not account for lm + trie load time when bench marking

Reply:
OK, I'll change it.